### DISCUSSION TITLE
视频语料标注的工作重点及其众包专案

### DISCUSSION THREAD

**Discussion by Guanchishan** [Nang-muoi-sing/cdo-corpus#32] [General]
榕典[寻求训练数据发展TTS功能](https://github.com/MindongLab/yngdieng/issues/272)的需求，导致急需经过标注的优质音频语料来投喂训练算法。拟设2020年下半年音频语料标注的工作重点如下。

- 福州话线上课堂范文朗读
- 南仙茶摊朗读

- [每日词汇挑战](https://github.com/MindongLab/foochow-beginner-guide/issues/2#issuecomment-678730078)

- [讲有味](https://mp.weixin.qq.com/s/b9k-KPVPmV5imctGxGaq8w)

欢迎补充。

独力无以毕功。我们将起动众包专案，以推动本工作有效率地完成。

专案准备进展：

- [ ] 写好相关教程与规范（如ELAN使用教程、标注规范）

- [ ] 起草呼吁加入的文章

- [ ] 确定众包任务分配管道；如有必要，搭建众包平台

**Comment by Guanchishan** [comment]
语料库作用：

 - 帮助更多人掌握福州话的音标和汉字（生态系统的建立，长期有利）
 - 榕典的“现实例句” 数据来源
 - （这个很模糊，最好有具体学者的需求佐证）研究用途
- 输入法的词频数据
- TTS/语音识别的训练数据
- 众包录入有助于增加分辨效率，有些词有人听半天听不出，有人一下子听出

by ztl

**Comment by Guanchishan** [comment]
语料众包管道

> 郑天镭：最好是有像 google docs / 石墨文档 一样的音频转写工具（“石墨音频？”） 可以在某个timecode直接评论，at 其他人来听一下

**Comment by ztl8702** [comment]
补充：语料来源  我觉得《讲有味》的质量挺高的 https://mp.weixin.qq.com/s/b9k-KPVPmV5imctGxGaq8w

**Comment by Guanchishan** [comment]
> 补充：语料来源  我觉得《讲有味》的质量挺高的 https://mp.weixin.qq.com/s/b9k-KPVPmV5imctGxGaq8w

added.
今后有觉得好的可以直接加到一楼里。
